docs/reflections/buehler/reflexion-sprint2.md

```diff
@@ -0,0 +1,86 @@
+2. **Dashboard-Funktionalität**: Nur die Anzeige von Transaktionen wurde umgesetzt, während das Löschen von Transaktionen noch fehlt.
+3. **Abgeschlossene Tickets**: Einige Tickets blieben offen, da Teammitglieder teilweise fehlten (Yannick fehlte zwei Wochen, Leon eine Woche).
+4. **Langsame Fortschritte**: Durch personelle Engpässe und technische Schwierigkeiten sind wir langsamer vorangekommen als erwartet.
+
+---
+
+### Gruppendynamik
+
+**Wie war die Zusammenarbeit im Team?**
+Die Zusammenarbeit war insgesamt sehr gut. Wir haben die Aufgaben klar verteilt, sodass jeder gezielt an einem Thema arbeiten konnte. Das Team arbeitete strukturiert und effizient.
+
+**Auf was achtet ihr bei der Kommunikation, damit diese konstruktiv ist?**
+Wichtig war, Probleme direkt anzusprechen und gemeinsam nach Lösungen zu suchen. Fehler wurden nicht kritisiert, sondern im Team erklärt, damit jeder daraus lernen konnte. Die offene und respektvolle Kommunikation hat dazu beigetragen, dass wir effizient arbeiten konnten.
+
+**Was ist uns besonders gut gelungen, worauf sind alle richtig stolz?**
+Besonders stolz sind wir auf das erfolgreiche Deployment in AWS. In anderen Teams gab es viele Fehler und Herausforderungen beim Deployment, während wir diesen Schritt vergleichsweise reibungslos geschafft haben.
+
+---
+
+### Lernzuwachs
+
+**Was kann ich nun mehr als vorher?**
+Ich habe gelernt, wie wichtig **Linting** und **Code-Formatierung** für die Lesbarkeit und Wartbarkeit des Codes sind. Ausserdem kann ich nun die Applikation in AWS deployen, was ein grosser praktischer Fortschritt ist.
+
+**Dieses Thema vom Sprint ist besonders relevant? Begründen Sie.**
+Für mich ist **Linting und Formatierung** besonders relevant. Im Arbeitsalltag ist es oft schwierig, unformatierten Code zu lesen, was viel Zeit kostet. Linting verhindert ausserdem, dass Fehler unbemerkt in die Produktion gelangen.
+
+**Was ist mir besonders gut gelungen, worauf bin ich richtig stolz?**
+Ich bin besonders stolz darauf, dass wir uns konsequent an unsere Tickets und Epics gehalten haben. Durch klare Planung und Zielsetzung sind wir gut vorangekommen und haben die richtigen Prioritäten gesetzt.
+
+---
+
+### Lerneffekt
+
+**Was sollte beibehalten werden?**
+Die Einführung in neue Themen war hilfreich und gut strukturiert. Es sollte dabei bleiben, dass wir selbstständig arbeiten können, damit jeder sein Thema vertieft. Dadurch konnten wir effizienter vorankommen.
+
+**Was sollte geändert werden?**
+Einige Punkte könnten optimiert werden:
+1. **Kürzere Vorträge**: Die Einführungen könnten kompakter gehalten werden, um schneller ins praktische Arbeiten einzusteigen.
+2. **Docker-Einführung**: Das Erstellen des Dockerfiles hat viel Zeit gekostet, teilweise wegen unterschiedlicher Einstellungen. Es wäre hilfreich, hier weniger Zeit zu investieren oder eine simplere Lösung anzubieten, da Docker nicht immer bei jedem reibungslos funktioniert.
```

Comment:
Welchen Dockerfiles? Vom Devcontainer oder vom Dockerfile fürs Deployment? Den Devcontainer werde ich zeimlich sicher optional halten in Zukunft, Dockerfiles fürs Deployment nach AWS sind unverzichtbar.